vocabularies/Common.xml

```diff
@@ -113,6 +113,17 @@
         </Member>
       </EnumType>
 
+      <Term Name="Timezone" Type="Edm.String" AppliesTo="Property Parameter">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The point in time represented by the annotated property or parameter shall be presented in the given time zone"/>

```

Comment:
"shall be presented" is UI terminology. Please replace with "common" terminology.